### PR TITLE
add new coredns image version: v1.11.0

### PR DESCRIPTION
**Pull request by tukwila**
https://github.com/coredns/coredns/releases/tag/v1.11.0
```
# nerdctl pull coredns/coredns:1.11.0
docker.io/coredns/coredns:1.11.0:                                                 resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:cc3ebb05fbdba439d2d69813f162aa204b027098c8244fb3156e6e7c0f31c548:    done           |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:28cccdd25b4d971951d50fd6e08ab5fe9a894d04db03557f2713bdf0347e7ddc: done           |++++++++++++++++++++++++++++++++++++++| 
config-sha256:321c7cb4ac8b4bc06b4e71e1b2526972245292a8502dce49d98843a271b3ba20:   done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:caa886227134d11f25a4cf824034a597eb7ec611bccdfc2785f7f342ed3e2c27:    done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:b28ed2365b9cb98936f0420af9de5343264762eb3e20fed4989e22149176664d:    done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 22.6s                                                                    total:  122.5  (5.4 KiB/s)
```

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-coredns/images.yaml [hunk 1]
```diff
@@ -14,3 +14,4 @@
     "sha256:b82e294de6be763f73ae71266c8f5466e7e03c69f3a1de96efd570284d35bb18": ["v1.9.4"]
     "sha256:017727efcfeb7d053af68e51436ce8e65edbc6ca573720afb4f79c8594036955": ["v1.10.0"]
     "sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e": ["v1.10.1"]
+    "sha256:cc3ebb05fbdba439d2d69813f162aa204b027098c8244fb3156e6e7c0f31c548": ["v1.11.0"]
```
Comment (ialidzhikov) [inline k8s.gcr.io/images/k8s-staging-coredns/images.yaml:17]: You have to update the image in `registry.k8s.io/images/k8s-staging-coredns`, not in `k8s.gcr.io/images/k8s-staging-coredns/images.yaml`.

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

**Comment by stp-ip** [comment]
After the fix on the correct file being used this should be good to go. I uploaded said release to staging already.

File: k8s.gcr.io/images/k8s-staging-coredns/images.yaml (hunk 1 above)
Comment (tukwila) [reply k8s.gcr.io/images/k8s-staging-coredns/images.yaml]: yes, i corrected it. -:)

File: k8s.gcr.io/images/k8s-staging-coredns/images.yaml [hunk 2]
```diff
@@ -13,4 +13,4 @@
     "sha256:8e352a029d304ca7431c6507b56800636c321cb52289686a581ab70aaa8a2e2a": ["v1.9.3"]
     "sha256:b82e294de6be763f73ae71266c8f5466e7e03c69f3a1de96efd570284d35bb18": ["v1.9.4"]
     "sha256:017727efcfeb7d053af68e51436ce8e65edbc6ca573720afb4f79c8594036955": ["v1.10.0"]
-    "sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e": ["v1.10.1"]
+    "sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e": ["v1.10.1"]
```
Comment (ialidzhikov) [inline k8s.gcr.io/images/k8s-staging-coredns/images.yaml:16]: Drop your diff from this file.

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

File: k8s.gcr.io/images/k8s-staging-coredns/images.yaml (hunk 2 above)
Comment (tukwila) [reply k8s.gcr.io/images/k8s-staging-coredns/images.yaml]: yes, i did.

**Pull request merged**